config/cookie.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cookies without encryption
+    |--------------------------------------------------------------------------
+    |
+    | OctoberCMS encrypts/decrypts cookies by default. You can specify cookies
+    | that should not be encrypted or decrypted here. This is useful, for
+    | example, when you want to pass data from frontend to server side backend
+    | via cookies, and vice versa.
+    |
+    */
+
+    'unecryptedCookies' => [
```

Comment:
Typo: `unecryptedCookies` should be `unencryptedCookies` (typo was in my original comment)